style.css

```diff
@@ -471,7 +476,17 @@ h1 {
     align-self: center;
   }
```

Comment:
- [ ] While you make a great effort 😊 I see that you put in the desktop version the number of cards as four, but in the Figma Design you will see that the number of cards aligned horizontally is 3 cards, so please try to make the number of the cards horizontally in the desktop version 3 like the Figma Design 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/3b283902-fc76-47b8-895a-906b50e8f3aa)   |     ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/cf08f08d-2ea3-4abc-a33d-75dcf46d4a43)  |